instfiles/openrc/xrdp

```diff
@@ -0,0 +1,4 @@
+#!/sbin/openrc-run
+
+command="__BASE__/sbin/xrdp"
+pidfile="/run/${RC_SVCNAME}.pid"
```

Comment:
PID directory might be configured by XRDP_PID_PATH. I don't recommend hardcoding this.

Reply:
`XRDP_PID_PATH` appears to map to `${localstatedir}/run` in many Makefile.am files, so this could be achieved by expanding the `install-data-hook:`

Personally I'm not that keen on using `install-data-hook` in this way - the way the systemd files are done is cleaner to my mind. That is however outside the scope of this change.

Reply:
> Personally I'm not that keen on using install-data-hook in this way - the way the systemd files are done is cleaner to my mind. That is however outside the scope of this change.

I thought the same thing. I didn't mentioned that because that is out of scope so we're totally on the same page.

Reply:
After reading about these comments and reviewing the code, it seems like there are a couple of different ways the code is currently configuring itself for installs. I was following the install-data-hook pattern.  I'd volunteer to open a separate pull request to look at unifying all of the install logic to whatever you think is the best practice way to do it. Since I'm already adding some new platforms with new install files I could look at making them all common.  In any case, for the current pull request, do you want me to make the PID_PATH changes?

Reply:
I resolved the other conversation since we were talking about similar things.  I fixed some whitespace errors in the current commit.  Let me know what you'd like me to do regarding this pull request, is it acceptable as is or do you want me to make some additional changes?